README.adoc

```diff
@@ -249,6 +395,15 @@ go get -u github.com/kisielk/errcheck
 | `make docker_push_master`     | Pushes master image to Docker image hosting repository
 |===
 
+
+=== Using Postman to test the REST API
+
+You can install the https://www.getpostman.com/[Postman] tool which will be useful to call and test the REST API endpoints of this server.
```

Comment:
```suggestion
TIP: You can install the https://www.getpostman.com/[Postman] tool which will be useful to call and test the REST API endpoints of this server.
```

Reply:
The reason I suggest this change is because I agree with @craicoverflow's comments above. I think it's fine to suggest it as a tip but it's not part of the project so is not necessary for the user to use this tool to run the project.

Reply:
Also after reviewing the file in full I do think that we should remove the image. It could get outdated quickly and it's not a tool that we are providing. The link out to postman should be sufficient.

Reply:
removed the topic and added as a TIP where has the curl requested.